.github/workflows/openqa_fullstack.yml

```
@@ -0,0 +1,21 @@
+---
+name: openQA Fullstack
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+jobs:
+  fullstack:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/ci/containers/base:latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Make project folder user-writable
+        run: sudo chown -R $USER ../../os-autoinst
+      - name: Clone openQA
+        run: git clone https://github.com/os-autoinst/openQA.git ../openQA
+      - name: Install dependencies
+        run: (cd ../openQA; bash -x tools/ci/build_cache.sh)
+      - name: Build os-autoinst
+        run: bash -x tools/ci/build_autoinst.sh
+      - name: Run unit tests
+        run: (cd ../openQA; tools/ci/run_unit_tests.sh fullstack)
```

I suggest

```suggestion
        run: make -C ../openQA test-fullstack
```

because all the additional parameters set by tools/ci/run_unit_tests.sh are likely not useful here, like saving in xunit compatible format and with coverage analysis. That's all additional overhead which we don't use here unless you upload the coverage and XUnit files as GHA artifacts